1_feature_engineering/1_prepare_inputs.py

```diff
@@ -23,16 +23,16 @@ def main():
     out_gs = 'gs://genetics-portal-dev-staging/l2g/{v}/features/inputs/'.format(v=args.version)
     manifest = {
         'study': {
-            'in': 'gs://genetics-portal-dev-staging/v2d/220208/studies.parquet',
+            'in': 'gs://genetics-portal-dev-staging/v2d/220210/studies.parquet',
```

Comment:
This study table is not up to date with the one being used in the rest of the pipeline: `gs://genetics-portal-dev-staging/v2d/220401/studies.parquet`